controllers/nstemplateset/client.go

```diff
@@ -0,0 +1,53 @@
+package nstemplateset
+
+import (
+	"github.com/codeready-toolchain/api/api/v1alpha1"
```

Comment:
could be aliased to `toolchainv1alpha1` like everywhere else

Reply:
done cf2f848